parlai/tasks/coco_caption/agents.py

```diff
@@ -11,10 +11,6 @@
 from .build_2017 import build as build_2017
 from .build_2017 import buildImage as buildImage_2017
 
-try:
-    import torch  # noqa: F401
```

Comment:
so we don't need torch here anymore right?

Reply:
Yes, as stephen mentioned, I removed the import torch statements ending in ```# noqa: F401``` other than in one place (where it was needed)